mlflow/mlflow
# Support compute specification when deploying SageMaker models

* This PR allows users to specify the type and quantity of SageMaker-compatible EC2 instances on which to deploy a model

**Test Strategy**

This PR is tested by manually evaluating the following cases via the Python API and CLI:

1. Deploy a model without specifying a non-default instance type or instance count. 
**Expected result**: The model should be deployed using the default instance type (`ml.m4.xlarge`) and instance count (`1`).

2. Deploy a model, specifying a non-default instance type. 
**Expected result**: The model should be deployed using the specified instance type and the default instance count (`1`).

3. Deploy a model, specifying a non-default instance count. 
**Expected result:** The model should be deployed using the default instance type (`ml.m4.xlarge`) and the specified instance count.

4. Deploy a model, specifying both a non-default instance type and a non-default instance count.
**Expected result**: The model should be deployed using the specified instance type and count.
